### PR TITLE
Enhance iOS compilation guide

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**
Give more detailed information on how to compile Jamulus for iOS.
@ngocdh could you please check if it’s ok?

**Context: Fixes an issue?**

No. Related to: https://github.com/jamulussoftware/jamuluswebsite/pull/643

**Does this change need documentation? What needs to be documented and how?**
No. 

**Status of this Pull Request**

Waiting on confirmation that it works as expected 
**What is missing until this pull request can be merged?**

Actual test If the steps are enough for a tech savvy person to get Jamulus running on iOS 
## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I tested my code and it does what I want
- [x] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#source-code-consistency) 
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I've filled all the content above

### CODE REVIEW

File: COMPILING.md [hunk 1]
```diff
@@ -141,10 +141,17 @@ Schemes:
...
 
 ## iOS
-* Make sure to install qt5 with the Qt Installer (not homebrew), and explicitly select iOS when choosing the Qt version
-* Run `/path/to/qt/5.15.2/ios/bin/qmake -spec macx-xcode Jamulus.pro`
-* Open the generated .xcodeproject in XCode, check the Signing & Capabilities and configure a team
-* To run on a iOS device, build and run on the device, you'll have first trust your developer profile in device's Settings.
+
+1. Install [Xcode from the Mac AppStore](https://apps.apple.com/us/app/xcode/id497799835?mt=12)
+2. [Download and install qt5 with the Qt Installer](https://www.qt.io/download) (not homebrew). Explicitly select iOS when choosing the Qt version
+3. Navigate to the folder of the Jamulus source code via terminal and run `/path/to/qt/5.15.2/ios/bin/qmake -spec macx-xcode Jamulus.pro` to generate an .xcodeproject file
```
Comment (gilgongo) [inline COMPILING.md:147]:
```suggestion
3. Go to the folder of the Jamulus source code via terminal and run `/path/to/qt/5.15.2/ios/bin/qmake -spec macx-xcode Jamulus.pro` to generate an .xcodeproject file
```

File: COMPILING.md [hunk 2]
```diff
@@ -141,10 +141,17 @@ Schemes:
...
+4. Open the generated .xcodeproject in Xcode
+5. Navigate to the Signing & Capabilities tab and fix signing errors by setting a team. Xcode might tell you if you need to change anything else like the bundle id.
```
Comment (gilgongo) [inline COMPILING.md:149]:
```suggestion
5. Go to the Signing & Capabilities tab and fix signing errors by setting a team. Xcode might tell you if you need to change anything else like the bundle id.
```

File: COMPILING.md [hunk 3]
```diff
@@ -141,10 +141,17 @@ Schemes:
...
+6. Connect your device via USB or if you set it up via WiFi
```
Comment (gilgongo) [inline COMPILING.md:150]:
```suggestion
6. Connect your device via USB (or WiFi if you set it up for that)
```

File: COMPILING.md [hunk 4]
```diff
@@ -141,10 +141,17 @@ Schemes:
...
+7. Select your device next to the play button
+8. Compile and run Jamulus by clicking on the play button
+9. Before being able to start Jamulus on your device, you'll have trust your developer profile in device's Settings under General>Profiles & Device Management. For more information [see the guide by osxdaily](https://osxdaily.com/2021/05/07/how-to-trust-an-app-on-iphone-ipad-to-fix-untrusted-developer-message/)
```
Comment (gilgongo) [inline COMPILING.md:153]:
```suggestion
9. Before being able to start Jamulus on your device, you'll have trust your developer profile in the device's Settings under General>Profiles & Device Management. For more information [see the guide by osxdaily](https://osxdaily.com/2021/05/07/how-to-trust-an-app-on-iphone-ipad-to-fix-untrusted-developer-message/)
```

File: COMPILING.md [hunk 5]
```diff
@@ -141,10 +141,17 @@ Schemes:
...
+10. After a week you might need to restart from step 6 to be able to run Jamulus on iOS
```
Comment (gilgongo) [inline COMPILING.md:154]: Is this only if you aren't paying? I you mention that elsewhere?  So might it be:

```suggestion
10. After a week you might need to restart from step 6 to be able to run Jamulus on iOS, unless you are paying for the Apple developer programme.
```

**Comment by ann0see** [comment]
Yeah... I assumed most people don't pay for a dev account.

**Comment by ann0see** [comment]
@gilgongo ready?

**Review by gilgongo — APPROVED** [review]

**Pull request merged**